src/app/detail/ui/ActivityWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+'use client'
+
+import { LogResponse } from '@/app/api/activity-logs/schemas/LogResponseSchema'
+import { Box, Skeleton, Stack, Typography } from '@mui/material'
+import { Comments } from './Comments'
+import { ActivityLog } from './ActivityLog'
+import { CommentInput } from '@/components/inputs/CommentInput'
+import { ActivityType } from '@prisma/client'
+import { postComment, deleteComment } from '../[task_id]/[user_type]/actions'
+import { CreateComment } from '@/types/dto/comment.dto'
+import { fetcher } from '@/utils/fetcher'
+import useSWR from 'swr'
+
+export const ActivityWrapper = ({ token, task_id }: { token: string; task_id: string }) => {
+  const { data: activities, isLoading } = useSWR(`/api/tasks/${task_id}/activity-logs/?token=${token}`, fetcher, {
```

Comment:
As discussed in our standup about leaving tasks app open all day and it hitting our server at regular intervals throughout that time, we should disable SWR fetches when the tab is not in focus.

An easy way to do this is to set `refreshWhenHidden` to `true`.